packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -52,16 +55,30 @@ export const generateTestDataForSlug = async (
     vite,
     templateFilepaths
   );
-  const featuresConfig = getTemplatesConfig(templateModuleCollection);
-  const featuresConfigForEntityPages: FeaturesConfig = {
-    features: featuresConfig.features.filter((f) => "entityPageSet" in f),
-    streams: featuresConfig.streams,
-  };
-  const args = getCommonArgs(featuresConfigForEntityPages, projectStructure);
-  args.push("--slug", slug);
 
-  const parsedData = await spawnTestDataCommand(stdout, "yext", args);
-  return getDocumentByLocale(parsedData, locale);
+  const allParsedData: any[] = [];
+  templateModuleCollection.forEach(async (templateModuleInternal) => {
```

Comment:
You shouldn't need to do this. You can just pass the slug off of the templateModuleInternal into generateTestDataForPage

Reply:
Would that be better than simply putting it into entityPageSet and then allowing the CLI to pick it up from there?